packages/xo-server-sdn-controller/src/index.js

```diff
@@ -437,11 +451,12 @@ class SDNController extends EventEmitter {
 
       const newHost = find(this._newHosts, { $ref: host.$ref })
       if (newHost !== undefined) {
+        this._newHosts.splice(this._newHosts.indexOf(newHost), 1)
```

Comment:
Except if you really need high performances, try to keep your collections immutable:
```js
this._newHosts = this._newHosts.filter(_ => _.$ref !== host.$ref)
```